### PR TITLE
Added File & Folder Action

### PR DESCRIPTION
**Pull request by myazarc**
### create(token, path, [success], [error])

Create folder. See [details](https://tech.yandex.com/disk/api/reference/create-folder-docpage/).

```javascript
import { resources } from 'ya-disk';

const API_TOKEN = '1982jhk12h31iad7a(*&kjas';

resources.create(API_TOKEN, 'disk:/folderName', () => {
  // success
},() => {
  //error
});
```
### remove(token, path, permanently, [success], [error])

Delete file or folder. See [details](https://tech.yandex.com/disk/api/reference/delete-docpage/).

```javascript
import { resources } from 'ya-disk';

const API_TOKEN = '1982jhk12h31iad7a(*&kjas';

resources.remove(API_TOKEN, 'disk:/fileOrFolderName', false, () => {
  // success
},() => {
  //error
});

### CODE REVIEW

**Comment by myazarc** [comment]
@RomiC thanks.
i was created PR. i had to use it in my project. Your answer is late, that's why i had to create new package. I will create new PR.

sorry for my bad english :)

**Comment by RomiC** [comment]
Oh, my bad! Don't know I've missed it. No, don't create a new PR. I'll open and review this one.

File: README.md [hunk 1]
```diff
@@ -1,9 +1,13 @@
-# ya-disk [![Build Status](https://travis-ci.org/RomiC/ya-disk.svg?branch=master)](https://travis-ci.org/RomiC/ya-disk) [![codecov](https://codecov.io/gh/RomiC/ya-disk/branch/master/graph/badge.svg)](https://codecov.io/gh/RomiC/ya-disk) [![Greenkeeper badge](https://badges.greenkeeper.io/RomiC/ya-disk.svg)](https://greenkeeper.io/)
+# ya-disk2
+---
+Original Repo: https://github.com/RomiC/ya-disk
+Original Package: https://www.npmjs.com/package/ya-disk
+---
```
Comment (RomiC) [inline README.md]: Pls, return the header back.

File: README.md [hunk 2]
```diff
@@ -1,9 +1,13 @@
...
 
 Yandex Disk API library which provides some methods for working with Yandex.Disk service API. Each method present with each own independent function. Example:
 
 ```javascript
-import { info } from 'ya-disk';
+import { info } from 'ya-disk2';
```
Comment (RomiC) [inline README.md]: Pls, rename the source lib back

File: README.md [hunk 3]
```diff
@@ -191,12 +195,43 @@ import upload from '../lib/upload';
...
\ No newline at end of file
+```
+### File and Folder Actions
+
+### create(token, path, [success], [error])
+
+Create folder. See [details](https://tech.yandex.com/disk/api/reference/create-folder-docpage/).
+
+```javascript
+import { resources } from 'ya-disk2';
```
Comment (RomiC) [inline README.md]: `ya-disk2` -> `ya-disk`

File: package.json [hunk 4]
```diff
@@ -1,15 +1,16 @@
 {
-  "name": "ya-disk",
-  "version": "1.3.1",
+  "name": "ya-disk2",
+  "version": "1.3.2",
```
Comment (RomiC) [inline package.json]: Revert these changes back. I'll bump the version right after merging your code

File: package.json [hunk 5]
```diff
@@ -30,9 +31,9 @@
     "sinon": "^6.1.5"
   },
   "bugs": {
-    "url": "https://github.com/RomiC/ya-disk/issues"
+    "url": "https://github.com/myazarc/ya-disk2/issues"
   },
-  "homepage": "https://github.com/RomiC/ya-disk#readme",
+  "homepage": "https://github.com/myazarc/ya-disk2#readme",
```
Comment (RomiC) [inline package.json]: Pls, change all links to the original repo.

**Comment by myazarc** [comment]
i completed requested changes.

File: README.md [hunk 6]
```diff
@@ -1,4 +1,4 @@
-# ya-disk [![Build Status](https://travis-ci.org/RomiC/ya-disk.svg?branch=master)](https://travis-ci.org/RomiC/ya-disk) [![codecov](https://codecov.io/gh/RomiC/ya-disk/branch/master/graph/badge.svg)](https://codecov.io/gh/RomiC/ya-disk) [![Greenkeeper badge](https://badges.greenkeeper.io/RomiC/ya-disk.svg)](https://greenkeeper.io/)
+# ya-disk
```
Comment (RomiC) [inline README.md]: Pls, return the badges back

**Pull request merged**